CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 5.1
+ * Add auto_hints_cleanup_enabled to web documentation (CASSANDRA-19685)
```

Comment:
One minor comment, it seems doc updates don't need to go into CHANGES.txt. So we can remove it from here

Reply:
Just updated the PR removing that line